app/async_consumer.py

```diff
@@ -51,12 +51,12 @@ def connect(self):
             self._url = settings.RABBIT_URLS[server_choice]
 
             try:
-                logger.info('Connecting', amqp_url=self._url, attempt=count)
+                logger.info('Connecting', attempt=count)
                 return pika.SelectConnection(pika.URLParameters(self._url),
                                              self.on_connection_open,
                                              stop_ioloop_on_close=False)
             except pika.exceptions.AMQPConnectionError as e:
-                logger.error("Connection error", exception=e, amqp_url=self._url)
+                logger.error("Connection error", exception=e)
```

Comment:
Same as above - should add a bit of context about which connection the error refers to.